service/crt/service.go

```diff
@@ -30,7 +29,7 @@ const (
 	// resyncPeriod is the period for re-synchronizing the list of objects in k8s
 	// watcher. 0 means that re-sync will be delayed as long as possible, until
 	// the watch will be closed or timed out.
-	resyncPeriod time.Duration = 0
+	resyncPeriod time.Duration = time.Minute * 10
```

Comment:
We need to set a value here. 0 is just wrong. See https://github.com/giantswarm/cert-operator/issues/49